mcstatus/pinger.py

```diff
@@ -25,7 +25,8 @@ class RawResponseVersion(TypedDict):
         protocol: int
 
     class RawResponseDescriptionWhenDict(TypedDict, total=False):
-        text: Required[str]
+        text: str  # not present if `translation` isn't None, same with `translation` field:
+        translation: str  # not present if `text` is not None
```

Comment:
Let's not use double negatives for no reason, and there's no need to mention the same thing twice.
```suggestion
        text: str  # present only if `translation` is set
        translation: str
```

Reply:
Fixed in f5ed0f937eb3fc2a44b690fd07ebdd667d1a55cb.

Reply:
The grammar in the fix is a bit odd. The comment should be more like: `it's present only if `translation` is set` and the translation should be `same to the above field`, though the comment from the translation filed could probably even be removed entirely.

Reply:
`it's present only if translation is set` sounds like `present` is a verb, shouldn't it be an adjective? And imperative mood shouldn't be used because we only get an object from server, and never send it back, so you can't specify what values will be here.

> the translation should be `same to the above field`

Will do this when we will end discussion for first field.

Reply:
Please follow my above suggestion, your version simply isn't correct, I don't really feel like explaining every little detail about the english grammar here on GitHub...

Reply:
Fixed in [this force push](https://github.com/py-mine/mcstatus/compare/f5ed0f937eb3fc2a44b690fd07ebdd667d1a55cb..549dae9e22c1996fc7c6a151cc31817167bff21b).